README.md

```diff
@@ -81,11 +88,18 @@ To **generate .rst files** from the code package, run the following command from
 ```bash
 sphinx-apidoc -o docs/source/ gimodules
 ```
+You most likely need pandoc installed on the system itself first to build:
 
-To **build the documentation**, run the following command:
+```bash
+sudo apt install pandoc
+```
+
+Then, to **build the documentation**, run the following commands:
 
 ```bash
 cd docs
+sudo apt update
+pip install --break-system-packages -r requirements.txt
```

Comment:
Why do we use: 
https://pip.pypa.io/en/stable/cli/pip_install/#cmdoption-break-system-packages